app/api/houdini/api.rb

```diff
@@ -1,5 +1,5 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
-class Houdini::API < Grape::API
+class Houdini::Api < Grape::API
```

Comment:
I couldn't get things working without changing from API to the Api acronym. Seems fine.

Reply:
You might have success with 

```
ActiveSupport::Inflector.inflections(:en) do |inflect|
  inflect.acronym 'API'
end
```

Reply:
I tried, it caused other bugs. I figured it wasn't worth fighting and we're removing the Grape API stuff so it won't be an issue going forward.